x-pack/plugins/observability_solution/synthetics/server/synthetics_service/formatters/formatting_utils.ts

```diff
@@ -50,9 +50,10 @@ export const replaceStringWithParams = (
   return value as string | null;
 };
 
+export const SHELL_PARAMS_REGEX = /\$\{[a-zA-Z_][a-zA-Z0-9\._\-?:]*\}/g;
```

Comment:
Why did you export it?

Reply:
In the moment thinking of unit testing use case where someone writing a test would want to import it.. but now that I'm typing it I agree it's not needed, as it's encapsulated in the function.